### PR TITLE
Activate ci

### PR DESCRIPTION
**Pull request by 2maz**
Activates travis CI and fix for Ubuntu 18.04 to let the tests pass.
Also includes an update of the metainformation/documentation to illustrate how the Dockerfile can be used on your local system.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+```
+    docker run -it rock/base-numeric /bin/bash
+```
+
+## License
+
+This work is licensed under [LGPL v2 or later](https://opensource.org/licenses/LGPL-2.1)
+
+## Copyright
+Copyright (c) 2012-2019 Jakob Schwender and Contributors, DFKI GmbH Robotics Innovation
```
Comment (doudou) [inline README.md]: If you really are big on having a line like this, you would at least have to add the BIR from Bahia (Joao Neto Tue May 10 12:49:19 2016 +0200 added the Savitzky Golay filter). I would suggest dropping the field altogether, or replacing it with something like

~~~
This project does not have a copyright transfer policy. Copyright is therefore retained by each individual author and/or their employers. See emails in git log for the complete list of authors.
~~~

**Comment by doudou** [comment]
If we want to have a standardized CI structure, I would personally prefer putting the files in `.ci` instead of in `test/ci`. Alternatively, just `ci/` at the toplevel (but I'd rather have it hidden under `.ci`)

File: README.md (hunk 1 above)
Comment (2maz) [reply README.md]: Not a big one, but IMO explicit mention is still better. I add the BIR

**Comment by 2maz** [comment]
> If we want to have a standardized CI structure, I would personally prefer putting the files in `.ci` instead of in `test/ci`. Alternatively, just `ci/` at the toplevel (but I'd rather have it hidden under `.ci`)

I have no strong preferences here, so moving to .ci

File: README.md (hunk 1 above)
Comment (doudou) [reply README.md]: > Not a big one, but IMO explicit mention is still better. I add the BIR

Suit yourself. It's just that explicit copyright notices are usually out-of-date (and therefore useless).

**Pull request merged**